cmd/aks-periscope/aks-periscope.go

```diff
@@ -56,16 +71,20 @@ func main() {
 			if err != nil {
 				log.Printf("Collector: %s, export data failed: %+v\n", c.GetName(), err)
 			}
+
 			waitgroup.Done()
 		}(c)
 	}
 
 	waitgroup.Wait()
 
 	diagnosers := []interfaces.Diagnoser{}
-	diagnosers = append(diagnosers, diagnoser.NewNetworkConfigDiagnoser(dnsCollector, kubeletCmdCollector, exporter))
```

Comment:
💡 Question for doc reasons: why `Arc` will not require `NetworkConfigDiagnoser`?

Reply:
The main reasoning was that the kubeletCmdCollector will be turned off if we enable "arc mode". Since NetworkConfigDiagnoser depends on the kubeletCmdCollector, we would not need NetworkConfigDiagnoser.